app/controllers/agentAppointment/AgentRelationshipController.scala

```diff
@@ -206,12 +213,41 @@ class AgentRelationshipController @Inject()(
   }
 
   def checkAnswers(): Action[AnyContent] = authenticated.andThen(withAppointAgentSession).async { implicit request =>
-    Future.successful(Ok(checkYourAnswers(request.sessionData)))
+    Future.successful(Ok(checkYourAnswers(submitAgentRelationship, request.sessionData)))
+  }
+
+  def submitAppointmentChanges(): Action[AnyContent] = authenticated.andThen(withAppointAgentSession).async {
```

Comment:
are we having only one endpoint in the frontend that does both? I am not 100% sure. that leaves us protected and makes the testng complicated. would it be possible to split this up making our testing jobs easier.

Reply:
golden rule, one endpoint one function.

Reply:
it does only do one thing, as per design 😄

Reply:
i do agree that we could do it differently. I will create separate methods for appoint and revoke

Reply:
> it does only do one thing, as per design 😄

the design say it APPOINT's and REVOKES which are at a base level two different functions.